src/moneypenny/moneypenny.py

```diff
@@ -5,32 +5,30 @@
 ]
 
 import asyncio
-
-from aiohttp import web
-from aiojobs import create_scheduler
-from aiojobs._job import Job
-
-from typing import List, Dict, Any
-
+import datetime
 import json
 import random
+from typing import Any, Dict, List
+
 import structlog
+import yaml
+from aiohttp import web
+from aiojobs import create_scheduler
 
 from .config import Configuration
+from .errors import (
+    CatGotYourTongueError,
+    NoMrBondIExpectYouToDie,
+    NonsensicalOrderError,
+)
 from .kubernetes import KubernetesClient
 from .singleton import Singleton
-from .types import CatGotYourTongueError, NonsensicalOrderError
-
-# These should be supplied as ConfigMaps
-M="/opt/lsst/software/moneypenny/config/m.yaml"
-QUIPS="/opt/lsst/software/moneypenny/config/quips.txt"
 
 logger = structlog.get_logger(__name__)
 
 
 class Moneypenny(metaclass=Singleton):
```

Comment:
Yeah don't do this, there's a whole post about how aiohttp is opinionated against singletons, so that's why there's init thing.  https://docs.aiohttp.org/en/stable/web_advanced.html#data-sharing-aka-no-singletons-please

Either way you don't need to worry about creating too many of those with special class sugar

Reply:
OK, Singleton dumped.